wallet/src/repository/wallet/state.rs

```diff
@@ -51,6 +51,8 @@ pub struct State {
     pub pending_addresses_by_path: HashMap<String, Arc<model::Address>>,
     /// List of pending blocks with state snapshots waiting to be confirmed
     pub pending_blocks: HashMap<String, StateSnapshot>,
+    /// List of pending dr movements, waiting to be confirmed with a superblock
```

Comment:
Can you give names to the types? This is a map of (dr_pointer) to (pending_block_hash, index). The index can be used to index pending_movements, which is a map of (pending_block_hash) to (Vec<BalanceMovement).

Reply:
of course